fe/src/components/Tab/HostTab.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <v-app  height="30px" width="5px">
```

Comment:
you may remove this and change it into container since I would duplicate in the app.vue file